docs/customizing-the-library/creating-custom-fields-and-widgets.md

```diff
@@ -1,5 +1,7 @@
 # Creating custom fields and widgets
 
+You can customize the base library in order to build web-based forms using React and the JSON Schema standard.
```

Comment:
I think perhaps the more specific use case here is: "You can customize the base library in order to satisfy the unique requirements of your particular form." Or something like that. Not that it just lets you build web-based forms, because the base library lets you do that, but so that you can do specific things that aren't required in the base library. It's a little reductive, but at least it's more accurate. Let me know if that makes sense!

Reply:
Yeah, thank you, I tried to bulk this up but was going to ask you for more info. This is great!